env/env_posix.cc

```diff
@@ -110,7 +110,7 @@ class PosixDynamicLibrary : public DynamicLibrary {
       return Status::OK();
     } else {
       char* err = dlerror();
-      return Status::NotFound("Error finding symbol: " + sym_name, err);
+      return Status::InvalidArgument("Error finding symbol: " + sym_name, err);
```

Comment:
This change is not related to current OBjectRegistry right?

Reply:
Removed from this PR